pkgs/top-level/all-packages.nix

```diff
@@ -14252,13 +14252,11 @@ with pkgs;
 
   haskell = callPackage ./haskell-packages.nix { };
 
-  # Please update doc/languages-frameworks/haskell.section.md, “Our
-  # current default compiler is”, if you bump this:
```

Comment:
This has been moved to https://haskell4nix.readthedocs.io/nixpkgs-users-guide.html, which still says 8.10.x. It would probably be better not to hardcode the current default in the docs and instead give instructions on finding out the default compiler version.

Reply:
Well, we will bring the docs back into the nixpkgs manual really soon (TM), then updating this will make sense again.